src/diet.ts

```diff
@@ -386,80 +290,200 @@ export function computeDiet(): [MenuItem[], number][] {
-      const firstValue =
-        value + meatDrop * Math.min(embezzlers, effectDuration) * 750 - (useMayo ? MPA : 0);
-      menu.push(new MenuItem(item, { maximum: 1, additionalValue: firstValue }));
-    }
-  }
+function copiers() {
+  const embezzlerDifferential = 25000 - 7500;
+  const alreadyGregarious =
+    get("beGregariousCharges") > 0 ||
+    (get("beGregariousFightsLeft") > 0 &&
+      get("beGregariousMonster") === $monster`Knob Goblin Embezzler`);
+
+  return [
+    new MenuItem($item`Extrovermectin™`, {
+      additionalValue: expectedGregs() * embezzlerDifferential,
+      maximum: alreadyGregarious ? 0 : 1,
+      note: "first gregarious",
+    }),
+  ];
+}
 
-  // We don't have a property to check if nothing has been eaten, so use this hack.
-  // This will fail in the rare case where someone has eaten non-PVPable food
-  // and then cleansed back down to 0.
-  if (
-    have($item`spaghetti breakfast`) &&
-    myFullness() === 0 &&
-    get("_timeSpinnerFoodAvailable") === "" &&
-    !get("_spaghettiBreakfastEaten")
-  ) {
-    menu.push(new MenuItem($item`spaghetti breakfast`, { maximum: 1 }));
-  }
+function countCopies(diet: [MenuItem[], number][]) {
+  const firstGregarious = diet.some((menuOption) =>
+    menuOption[0].some(
+      (menuItem) => menuItem.item === $item`Extrovermectin™` && menuItem.note === "first gregarious"
+    )
+  );
 
-  // Only use our astral pilsners if we're ascending. Otherwise they're good for shotglass.
-  if (have($item`astral pilsner`) && globalOptions.ascending) {
-    menu.push(
-      new MenuItem($item`astral pilsner`, {
-        maximum: availableAmount($item`astral pilsner`),
-      })
+  return (firstGregarious ? 1 : 0) * expectedGregs();
+}
+
+/**
+ * Generate a potion diet that has entries
+ * @param embezzlers number of embezzlers expected to be encountered on this day
+ * @param turns number of turns total expecte
+ */
+export function potionMenu(baseMenu: MenuItem[], embezzlers: number, turns: number): MenuItem[] {
+  function limitedPotion(input: Item | Potion, limit?: number, options = {}) {
+    let potion = input instanceof Item ? new Potion(input) : input;
+    if (itemType(potion.potion) === "food" && MayoClinic.installed()) {
+      potion = potion.doubleDuration();
+    }
+    return potion.value(embezzlers, turns, limit).map(
+      (tier) =>
+        new MenuItem(potion.potion, {
+          ...options,
+          maximum: tier.quantity,
+        })
     );
   }
+  function potion(potion: Item | Potion, options = {}) {
+    return limitedPotion(potion, undefined, options);
+  }
+
+  return [
+    ...baseMenu,
+    ...copiers(),
+
+    // FOOD POTIONS
+    ...potion($item`jumping horseradish`),
+    ...potion($item`jumping horseradish`),
+    ...potion($item`tempura cauliflower`),
+    ...potion($item`sea truffle`),
+    ...potion($item`tempura broccoli`),
+    ...limitedPotion(
+      $item`Dinsey food-cone`,
```

Comment:
For some reason my garbo isn't getting the food-cones automatically.

Reply:
Oh, looks like it's because it hasn't opened the zone yet.

Reply:
This means that we need to account for nobarf when we do that, too